lib/internal/test_runner/runner.js

```diff
@@ -487,29 +487,31 @@ function run(options = kEmptyObject) {
   }
 
   const root = createTestTree({ __proto__: null, concurrency, timeout, signal });
-  let testFiles = files ?? createTestFileList();
+  if (process.env.NODE_TEST_CONTEXT === undefined) {
```

Comment:
```suggestion
  if (process.env.NODE_TEST_CONTEXT !== undefined) {
  	return root.reporter;
  }
```
nitpick: avoid extra indentation